docs/development/language-features.md

```diff
@@ -0,0 +1,208 @@
+# Contributing to a Language
+
+Improving Yomitan's features for the language(s) you are interested in is pretty simple, and a great way to help yourself and others. This guide will help you get started.
+
+## Adding a Language
+
+<img style="float: right; margin-left: 20px; " src="../../img/language-dropdown.png">
+
+If your language is not already available in the Language dropdown, here is how you can add it with just a few lines. As an example, we'll use [PR #913](https://github.com/themoeway/yomitan/pull/913/files), where a first-time contributor added Dutch.
+
+```js
+// language-descriptors.js
+{
+    iso: 'nl',
+    name: 'Dutch',
+    exampleText: 'lezen',
+    textPreprocessors: capitalizationPreprocessors
+}
+```
+
+1. Look up the ISO 639-1 code for your language. If it is a rarer language it might not have a ISO-639-1 code - if that's the case, use ISO 639-3.
+2. Place your language in the `languageDescriptors` array in `language-descriptors.js`. The languages are sorted alphabetically by ISO code.
+3. The example text is usually some form of the verb "to read" in your language, but it can be any example you feel is good. This will be shown in the preview popup on the settings page.
+4. If your language uses the Latin or Cyrillic script, or another script with capitalization, you will want to use the `capitalizationPreprocessors`. We'll cover this part in more detail a bit later. The `textPreprocessors` field can also be left out.
+
+When in doubt, look at the other languages in the file for ideas. The same applies to `language-descriptors.d.ts`:
+
+```ts
+// language-descriptors.d.ts
+nl: {
+  pre: CapitalizationPreprocessors;
+}
+```
+
+This is just for some type safety. The first key is the ISO code. Most languages will then only have a `pre` key (the other one is `post`), and its value is the type of text preprocessors you used in `language-descriptors.js`. Use the TypeScript operator `&` as needed. If you didn't use any text preprocessors, you can set the value to `Record<string, never>`.
+
+That's it! Your language should now be selectable from the dropdown, and may work perfectly fine already. For more advanced features, read on.
```

Comment:
Maybe add a link to yomitan dictionaries? Hard to know if your language works without testing with dictionary